RadClass/RadClass.py

```diff
@@ -0,0 +1,205 @@
+import numpy as np
+import pandas as pd
+import time
+import logging
+import progressbar
+
+import RadClass.DataSet as ds
+
+
+class RadClass:
+    '''
+    Bulk handler class. Contains most functions needed for processing data
+        stream files and pass along to an analysis object.
+
+    Attributes:
+    running: indicates whether the end of a file has been reached and thus end.
+    stride: The number of seconds (which equals the number of indexes
+        to advance after analyzing an integration interval. e.g. if
+        stride = 3, then the next integration period will setp 3 seconds ahead.
+        If stride = integration, all observations will be analyzed once.
+    integration: The number of seconds (which equals the number of indexes)
+        indexes) to integrate the spectrum over. e.g. if integration = 5, then
+        the next 5 rows (corresponding to the next 5 seconds) of data will be
+        pulled and summed/integrated to analyze.
+    datapath: HDF5 group/subgroup path to dataset, including prefix,
+        node ID, and analysis element. e.g. '/path/to/dataset'
+    filename: The filename for the data to be analyzed.
+    TODO: Make node and filename -lists- so that multiple nodes and files
+        can be processed by the same object.
+    store_data: boolean; if true, save the results to a CSV file.
+    cache_size: (WIP) optional parameter to reduce file I/O and therefore
+        increase performance. Indexes a larger selection of rows to analyze.
+        If not provided (None), cache_size is ignored (equals integration).
+    labels: list of dataset name labels in this order:
+        [ live_label: live dataset name in HDF5 file,
+          timestamps_label: timestamps dataset name in HDF5 file,
+          spectra_label: spectra dataset name in HDF5 file ]
+    '''
+
+    running = True
```

Comment:
Does this need to be a member variable? It seems like it's only used as a local variable in one method